src/Collector/Stream/FilesystemStreamProxy.php

```diff
@@ -105,8 +105,8 @@ public static function unregister(): void
     private function isIgnored(): bool
     {
         $backtrace = debug_backtrace();
-        return BacktraceIgnoreMatcher::isIgnoredByClass($backtrace, self::$ignoredClasses)
-            || BacktraceIgnoreMatcher::isIgnoredByFile($backtrace, self::$ignoredPathPatterns);
+        return BacktraceMatcher::byClass($backtrace[3], self::$ignoredClasses)
```

Comment:
What could either `byClass` or `byFile` method mean? 
Can't see any improvements here

Reply:
Backtrace match (class name) by file/class (method name).

"Ignore" doesn't matter here.

Reply:
> * if you rename a class rename it instead of removing and adding a new one

That is what GitHub decided, not me :)

> * a class method must be a verb
> * better name for the methods should contain `contains` / `includes` / `intersects` 'cause it contains searching operation

I don't think it makes sense in this case, because all methods perform single  action. But it doesn't matter to me. Is "matchByFile()“ OK?

> * the matcher now depends on the proxy implementation, which was independent before and could be used in any other flows

On the contrary, now the matcher could be used in any other flows, because it not depends from internal structure of calling class (It depends on which item of backtrace to use for matching).

Reply:
Methods are renamed.